modules/statics/src/base.ts

```diff
@@ -1521,7 +1521,7 @@ export enum UnderlyingAsset {
   ZCO = 'zco',
   ZECBEAR = 'zecbear',
   ZECBULL = 'zecbull',
-  ZETACHAIN = 'zetachain',
```

Comment:
What are the consequences of removing this ENUM value downstream? Can we please check

Reply:
I think the consequences of removing is anyone, because i'm updating this one. But i need to update in Indexer and BGMS also.